multisig/proposals/opsepolia/vip-001/index.ts

```diff
@@ -0,0 +1,183 @@
+import { NETWORK_ADDRESSES } from "src/networkAddresses";
+import { makeProposal } from "src/utils";
+
+const { opsepolia } = NETWORK_ADDRESSES;
+
+const CHAINLINK_BTC_FEED = "0x3015aa11f5c2D4Bd0f891E708C8927961b38cE7D";
+const CHAINLINK_ETH_FEED = "0x61Ec26aA57019C486B10502285c5A3D4A4750AD7";
+const CHAINLINK_USDC_FEED = "0x6e44e50E3cc14DD16e01C590DC1d7020cb36eD4C";
+const CHAINLINK_USDT_FEED = "0xF83696ca1b8a266163bE252bE2B94702D4929392";
+const CHAINLINK_OP_FEED = "0x8907a105E562C9F3d7F2ed46539Ae36D87a15590";
+
+const WBTC = "0x149e3b3bd69f1cfc1b42b6a6a152a42e38ceebf1";
```

Comment:
I think it's better if we use mock tokens under our control for WBTC, USDC, USDT and OP, instead of using tokens deployed and managed by third parties. We would need to be able to mint these tokens, for example

Reply:
Updated mock token addresses, 78d18fc3e5a42474289fd7aba2f13ecd26d67456